README.md

```diff
@@ -4,25 +4,8 @@ WebLogic Monitoring Exporter
 [![Build Status](https://travis-ci.org/oracle/weblogic-monitoring-exporter.svg?branch=master)](https://travis-ci.org/oracle/weblogic-monitoring-exporter)
 
 The WebLogic Monitoring Exporter uses the [WLS RESTful Management API](https://docs.oracle.com/middleware/1221/wls/WLRUR/overview.htm#WLRUR111) to scrape runtime information and then exports [Prometheus](http://prometheus.io)-compatible metrics.
-It is deployed as a web application in a WebLogic Server (WLS) instance, version 12.2.1 or later, typically, in the instance from which you want to get metrics.
-
-## Downloading the release
-
-You can find all the releases on the [Releases page](https://github.com/oracle/weblogic-monitoring-exporter/releases).
-
-To download the web application `wls-exporter.war` and put your configuration file into the WAR, download the `getXXX.sh` script, which is provided with each release and also can be downloaded from the Releases page, and then run:
-
-```
-bash getXXX.sh <your-config-file>
-```
-
-## Building from source
-
-Use `mvn install` to build the web application. This will create `wls-exporter-<version>`, where _version_
-is the Maven-assigned version number.
-
-Adding `-Dconfiguration=<some-config-file>` will insert the specified configuration as its default and remove
-the version number to simplify deployment to WebLogic Server.
+It is available both as a [web application](#web-application) to be deployed to the WebLogic Server (WLS) instance, version 12.2.1 or later, from which you want to get metrics, 
+and also as a [sidecar](#sidecar), which runs as its own process, and which is configured to contact a WLS instance.
```

Comment:
Can there be a table or similar that documents the trade-offs between the two approaches?  For example, the side-car requires maintaining a separate process with its own external port, potentially in a separate container if docker/k8s is in the mix, but WKO conveniently takes care of that for you automatically assuming WKO is in the mix. And I assume the main reason to keep using the web-app  is if you want to avoid upgrade issues from older versions (which only supported the web-app).  Also, it looks like they have different APIs for configuration.  Finally, the web-app requires altering WL configuration to deploy, or using sit-cfg.

Reply:
The main reason to use the web app is for use _outside_ of the operator, where it is simpler to deploy a WAR, whereas the sidecar is intended for use with the operator, which will do the configuration. I can certainly make that more clear in the docs.